Supporting Documents/BS_SD.adoc

```diff
@@ -1283,6 +1282,138 @@ In this scenario, at least 30,000 imposter transactions shall be conducted with
+
+[loweralpha]
+. Quality assessment test configuration
++
+[arabic]
+.. TOE Reference
++
+Information that uniquely identifies the TOE shall be reported. <<BIOPP-Module>> is intended to be used with the Base-PP and reference for the computer can be used as the TOE reference only if the reference for the computer also uniquely identifies the biometric system embedded in the computer
++
+Modification to the TOE for performance testing, if any, shall be reported (e.g. the TOE is modified to export biometric data for reporting quality output). The rationale that such modification does not affect the TOE performance shall also be provided. For example, the developer may claim that the performance is not affected because modified code is not executed during biometric verification or the developer may run regression tests to verify that modification does not change the result of verification (e.g. similarity score).
+.. TOE configuration
++
+Any configurable parameters or settings of the TOE that may affect the performance shall be reported. The value of each parameter set for the testing shall also be provided. For example, if the threshold (e.g. decision threshold and image quality threshold) is configurable by users, the value of the threshold set for the testing shall be reported.
+.. How quality is measured
++
+<<ISO29794-1, ISO/IEC 29794-1>> defines quality as the "degree to which a biometric sample fulfills specified requirements for a targeted application". The way quality is measured using character, fidelity and utility as defined in <<ISO29794-1, ISO/IEC 29794-1>> shall be reported.
+.. Type of quality algorithm
++
+The type of algorithm used to determine the quality of the sample.
+.. Quality assessment test tools
++
+Information that uniquely identifies all testing tools (e.g. SDK) used for the quality assessment testing shall be reported.
+
+[loweralpha, start=2]
+. Result of the quality testing
++
+The quality assessment report shall provide the following items from the result of testing:
++
+[arabic]
+.. Test period and location
++
+Timeline for the quality assessment testing (samples may be collected over multiple sessions if necessary) and location of testing shall be reported.
+.. Modality used for biometric verification
++
+The quality assessment testing shall be done for all modalities selected in FIA_MBV_EXT.1. The results of testing for each modality shall be reported separately.
+.. Number of test subjects and samples
++
+* Test subjects
++
+Number of test subjects who participated in the testing shall be reported.
+* Samples
++
+Number of samples collected for each body part and total number of samples collected from all test subjects shall be reported.
++
+Note all test subjects may not generate the samples successfully and total number of samples may be less than (number of test subjects) × (number of body parts of a test subject) × (number of samples collected for each body part).
```

Comment:
Not sure if typo, so please check. Also trying to avoid "body part."

```suggestion
Not all test subjects may generate the samples successfully and total number of samples may be less than (number of test subjects) × (number of modalities of a test subject) × (number of samples collected for each modality).
```